components/item/MediaActionBar.vue

```diff
@@ -82,7 +82,7 @@
       </b-col>
     </b-row>
     <b-row
-      v-else
+      v-show="showShareButtons"
```

Comment:
Could you update me in why you have changed this into a v-show?

Reply:
v-if removes the element from the DOM, so when it does become finally visible, the tabbing index messes up, stopping the user from tabbing through the 3 social buttons. Using v-show, keeps the element in the DOM, but hidden from the user. Maybe we should encourage this more throughout, as I am sure its probably better for screen readers too.